via/views/view_video.py

```diff
@@ -9,8 +10,10 @@
 
 
 @view_config(renderer="via:templates/view_video.html.jinja2", route_name="youtube")
-@use_kwargs({"url": fields.Url(required=True)}, location="query")
-def youtube(request, url):
+@use_kwargs(
+    {"url": fields.Url(required=True)}, location="query", unknown=marshmallow.INCLUDE
+)
+def youtube(request, url, **kwargs):
```

Comment:
Added `unkwown=mardhsmallow.INCLUDE` so that unknown query params get passed through. See <https://webargs.readthedocs.io/en/latest/advanced.html#setting-unknown> and <https://marshmallow.readthedocs.io/en/stable/quickstart.html#handling-unknown-fields>. Previously the schema was raising `ValidationError` if there were any query params other than `url`.

Any unknown query params get passed to the view as kwargs, hence the `**kwargs`.